Directory.Build.targets

```diff
@@ -3,18 +3,37 @@
 
   <!-- Add Roslyn analyzers NuGet to all projects -->
   <ItemGroup Condition=" '$(DisableRoslynAnalyzers)' != 'True' ">
-    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers">
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.3.0">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
   </ItemGroup>
 
-  <!-- Set Assets for NUnit3TestAdapter -->
+  <!-- NuGet Dependencies -->
   <ItemGroup>
-    <PackageReference Update="NUnit3TestAdapter">
+    <PackageReference Update="GitInfo"                                      Version="2.1.2" />
+    <PackageReference Update="HtmlAgilityPack"                              Version="1.11.30" />
+    <PackageReference Update="Irony"                                        Version="1.1.0" />
+    <PackageReference Update="Microsoft.Build.Framework"                    Version="17.3.2" />
+    <PackageReference Update="Microsoft.Build.Utilities.Core"               Version="17.3.2" />
+    <PackageReference Update="Microsoft.CodeAnalysis.CSharp"                Version="4.3.1" />
+    <PackageReference Update="Microsoft.CSharp"                             Version="4.7.0" />
+    <PackageReference Update="Microsoft.DotNet.GenAPI"                      Version="7.0.0-beta.22103.1" />
+    <PackageReference Update="Microsoft.NET.Test.Sdk"                       Version="17.5.0-preview-20221003-04" />
+    <PackageReference Update="Microsoft.NETFramework.ReferenceAssemblies"   Version="1.0.3" />
```

Comment:
This entry should not be needed, and was removed entirely in bf8d78fbd3bfcb29cc2e37fa84e275acf6c668d6.  (Which, yes, we're reverting, but it could have been removed when we dropped net472 support…)